python/aswfdocker/tests/test_builder.py

```diff
@@ -405,7 +481,7 @@ def test_builder_cli_conan(self):
         )
         self.assertTrue(
             cmds[2].endswith(
-                f"conan create {constants.CONAN_USER_HOME}/recipes/openexr openexr/2.3.0@aswftesting/vfx2019'"
+                f"conan create {os.path.join(self._docker_home, 'recipes', 'openexr')} openexr/2.3.0@aswftesting/vfx2019'"
```

Comment:
Good change to not build paths yourself. Could be interesting exercise to grep for `/` in the whole codebase (you may have already done that).